src/client/activation/migrateDataScienceSettingsService.ts

```diff
@@ -64,6 +65,12 @@ export class MigrateDataScienceSettingsService implements IExtensionActivationSe
                 k = 'xxxxxx.dataScience.jupyterServerType';
             }
 
+            // If the value contains references to python.dataScience.* commands, migrate those too
+            if (typeof val === 'string') {
+                // There may be multiple occurrences of commands in the object value
+                val = val.replace(/python\.dataScience\./gi, 'jupyter.');
```

Comment:
What happens if users already created the correct version of the settings.
Now we end up with duplicates.
E.g. if a user sees someone doesn't work, they have two choices:
* Rename existing settings
* Create new settings.

I'm concerned some may have created the new settings. & this could would not create duplicates.